script/staticLinkLibDownload.js

```diff
@@ -0,0 +1,83 @@
+const ENVIRONMENT = process.env.npm_lifecycle_event;
+
+if(ENVIRONMENT === "build") {
```

Comment:
This script is certainly running under `postinstall`.

These `if` states is necessary
.

Reply:
```suggestion
if(ENVIRONMENT === "build") {
```
excuse me, i am still a little bit confused about what u said, could u please explain more, thanks.
as i understand, the lib must be downloaded before the `build` process and after `install` process, so i write the code in the `postInstall` process. why u said the `build` if states is necessary ?

Reply:
Sorry, these `if` states are not necessary. You can consider that this script is certainly running under `postInstall`.